libs/template/file_test.go

```diff
@@ -0,0 +1,137 @@
+package template
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/databricks/cli/libs/filer"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestTemplateFileCommonPathForWindows(t *testing.T) {
+	if runtime.GOOS != "windows" {
+		t.SkipNow()
+	}
+	f := &fileCommon{
+		root:    `c:\a\b\c`,
+		relPath: "d/e",
+	}
+	assert.Equal(t, `c:\a\b\c\d\e`, f.Path())
+	assert.Equal(t, `d/e`, f.RelPath())
+}
+
+func TestTemplateFileCommonPathForUnix(t *testing.T) {
+	if runtime.GOOS != "linux" && runtime.GOOS != "darwin" {
```

Comment:
You can use `!= "windows"` for symmetry with the above case.

We can differentiate between Windows and not-Windows.

Reply:
done